core/forms.py

```diff
@@ -54,6 +54,8 @@ class EditProfileForm(ModelForm):
     """
     Edit Profile form.
     """
+    profile_photo = forms.ImageField(help_text='Upload profile photo', required=False)
```

Comment:
why not use field `photo` of model `Profile`

Reply:
there was a problem with using `ModelForm` field of `photo` from `profile` model. The `photo` field was always empty and the working solution to that was to introduce a separate form field.